lib/elixir/lib/kernel.ex

```diff
@@ -1206,13 +1228,22 @@ defmodule Kernel do
   @doc """
   Gets the element at the zero-based `index` in `tuple`.
 
+  It raises `ArgumentError` when `tuple` is empty,
+  or when `index` is a negative number or it is out of range of the tuple elements.
```

Comment:
In `elem({:a}, 5)` the index is out of range. Saying that the function will raise when the tuple is empty or the index is negative is redundant if you also say that it will raise on out of range.